criu/cr-restore.c

```diff
@@ -2051,6 +2051,17 @@ static int restore_root_task(struct pstree_item *init)
 	if (ret < 0)
 		goto out;
 
+	pr_info("Run restore asynchronous hook from criu master for external devices\n");
+	ret = run_plugins(RESTORE_ASYNCHRONOUS);
```

Comment:
I think it isn't a good name for this hook. It can be a bit more descriptive about the exact moment when it is called.